crt_portal/cts_forms/templates/forms/complaint_view/index/filter.html

```diff
@@ -1,4 +1,4 @@
-{% load static %}
```

Comment:
If there's no changes in this file (and the whitespace isn't necessary here), please omit from the PR as it makes subsequent merges cleaner.